app/config/permissions.neon

```diff
@@ -114,15 +114,22 @@ permissions:
 
     - allow: true
       resource: group
+      role: supervisor-student
```

Comment:
Just to let you know... this does not really do anything... You were probably just clarifying the permissions?

Reply:
Just a bit of defensive programming. And yes, second purpose is clarification.